torch/csrc/jit/tensorexpr/llvm_codegen.cpp

```diff
@@ -167,10 +167,10 @@ class LLVMCodeGenImpl : public IRVisitor {
   llvm::Type* Int8PtrTy_;
   llvm::Type* VoidTy_;
 
-  std::unordered_map<const Var*, int> varToArg_;
```

Comment:
We can keep `Var` to be `const` in this case for better coding.

Reply:
I wasn't sure about this. I wanted to keep this PR simple in that it removes const from all NNC IR. But, in this case, since it is just a key to a map and making it const did not affect any other code, I'm okay doing that. 

Updated.